### ISSUE TITLE
failed to install in China

### ISSUE THREAD

**Issue by bi-kai** [MayhemYDG/iltorb#41]
I cannot access AWS in China even with shadowsocket:
```
> node-pre-gyp install --fallback-to-build

node-pre-gyp ERR! Tried to download(undefined): https://node-iltorb.s3.amazonaws.com/iltorb/v1.3.4/node-v48-win32-x64.tar.gz
node-pre-gyp ERR! Pre-built binaries not found for iltorb@1.3.4 and node@6.10.3 (node-v48 ABI) (falling back to source compile with node-gyp)
gyp WARN install got an error, rolling back install
gyp ERR! configure error
gyp ERR! stack Error: tunneling socket could not be established, statusCode=407
gyp ERR! stack     at ClientRequest.onConnect (D:\node\www\node_modules\tunnel-agent\index.js:165:19)
gyp ERR! stack     at ClientRequest.g (events.js:292:16)
gyp ERR! stack     at emitThree (events.js:116:13)
gyp ERR! stack     at ClientRequest.emit (events.js:194:7)
gyp ERR! stack     at Socket.socketOnData (_http_client.js:394:11)
gyp ERR! stack     at emitOne (events.js:96:13)
gyp ERR! stack     at Socket.emit (events.js:188:7)
gyp ERR! stack     at readableAddChunk (_stream_readable.js:176:18)
gyp ERR! stack     at Socket.Readable.push (_stream_readable.js:134:10)
gyp ERR! stack     at TCP.onread (net.js:547:20)
gyp ERR! System Windows_NT 6.1.7601
gyp ERR! command "D:\\node\\anzhuang\\node.exe" "D:\\node\\www\\node_modules\\node-gyp\\bin\\node-gyp.js" "configure" "--fallback-to-build" "--module=D:\\node\\www\\node_modules\\iltorb\\build\\bindings\\encode.node" "--module_name=encode" "--module_path=D:\\node\\www\\node_modules\\iltorb\\build\\bindings" "--msvs_version=2015"
gyp ERR! cwd D:\node\www\node_modules\iltorb
gyp ERR! node -v v6.10.3
gyp ERR! node-gyp -v v3.6.1
gyp ERR! not ok
node-pre-gyp ERR! build error
node-pre-gyp ERR! stack Error: Failed to execute 'D:\node\anzhuang\node.exe D:\node\www\node_modules\node-gyp\bin\node-gyp.js configure --fallback-to-build --module=D:\node\www\node_modules\iltorb\build\bindings\encode.node --module_name=encode --module_path=D:\node\www\node_modules\iltorb\build\bindings --msvs_version=2015' (1)
node-pre-gyp ERR! stack     at ChildProcess.<anonymous> (D:\node\www\node_modules\iltorb\node_modules\node-pre-gyp\lib\util\compile.js:83:29)
node-pre-gyp ERR! stack     at emitTwo (events.js:106:13)
node-pre-gyp ERR! stack     at ChildProcess.emit (events.js:191:7)
node-pre-gyp ERR! stack     at maybeClose (internal/child_process.js:886:16)
node-pre-gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:226:5)
node-pre-gyp ERR! System Windows_NT 6.1.7601
node-pre-gyp ERR! command "D:\\node\\anzhuang\\node.exe" "D:\\node\\www\\node_modules\\iltorb\\node_modules\\node-pre-gyp\\bin\\node-pre-gyp" "install" "--fallback-to-build"
node-pre-gyp ERR! cwd D:\node\www\node_modules\iltorb
node-pre-gyp ERR! node -v v6.10.3
node-pre-gyp ERR! node-pre-gyp -v v0.6.36
node-pre-gyp ERR! not ok
```

Can you put that package on npmjs.com or somewhere else?

**Comment by oohnoitz** [comment]
Will look into moving the hosted pre-compiled binaries to GitHub when I have the time.

**Comment by oohnoitz** [comment]
Published `1.3.5`. We are now serving the pre-compiled binaries with GitHub releases instead of S3 now. This should resolve any issues where S3 is blocked.
